govc/tags/tags_helper/categories.go

```diff
@@ -0,0 +1,202 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tags
```

Comment:
We need this package to be outside of the govc directory, as it will be consumed outside of govc.  The rest APIs are referred to as `vAPI`, so let's move the `govc/tags/tags_helper` directory to `vapi/tags` at the top-level of the repo.